packages/cli/src/start.ts

```diff
@@ -88,7 +92,8 @@ export async function start(
 		});
 
 		const pair = await getPairAsync(secret);
-		env = new ProviderEnvironment(config, pair);
+		const authAccount = await getPairAsync(config.authAccount.secret);
```

Comment:
I don't think this should be secret for the providers cause that means they need to retain the secret in their env files. You can getPairAsync using an address only by doing 

```
getPairAsync( undefined, address )
```

and then you can use this pair to confirm signatures.